acl_loader/main.py

```diff
@@ -758,7 +759,7 @@ def incremental_update(self):
                 namespace_configdb.mod_entry(self.ACL_RULE, key, None)
 
         for key in existing_controlplane_rules:
-            if cmp(self.rules_info[key], self.rules_db_info[key]) != 0:
+            if operator.eq(self.rules_info[key], self.rules_db_info[key]) != 0:
```

Comment:
`operator.eq` will return a bool?

Reply:
@qiluo-msft Yes. The return type of this method is bool, it returns True if x is equal to y, False, otherwise.

Reply:
@qiluo-msft Thank you for your question, for python2, cmp returns 0 if x is equal to y, if not equal, the result is not 0, it should be -1 or 1. So, operator.eq's return result is opposite with cmp's results. I will update code to make the logic right and write unit test case as well.

Syntax:
cmp(a, b)
Parameters:
a and b are the two numbers in which the comparison is being done. 
Returns:
-1 if a<b
0 if a=b
1 if a>b

Reply:
I see your new iteration. To make it better, use `if not eq()`.

Reply:
@qiluo-msft Ye, agree, I updated it.